django/urls/resolvers.py

```diff
@@ -514,6 +499,12 @@ def _check_custom_error_handlers(self):
         return messages
 
     def _populate(self):
+
+        # Import the functionality of _callback_strs, lookup_str from admindocs.utils
+        from django.contrib.admindocs.utils import _callback_strs, lookup_str
+
+        callback_strs = _callback_strs()
```

Comment:
`django.urls` shouldn't depend on `django.contrib.admindocs`.

Instead, `django.urls` should provide a way for other apps to register callbacks with the resolver that the resolver will call.

Reply:
True, I had my doubts regarding this, and I found that moving the functionality of `_callback_strs` is not that much needed as it is used just once in the whole section and that to by function which is being called more than enough times.

And therefore I think it would be good if we don't move this variable to `django.contrib.admindocs`.

Reply:
Fixed!